### PR TITLE
Allow editing tickets in Escaping event

### LINKED ISSUE #1810: [BUG] Escape event
How to reproduce: 
Be a class (in my instance a special scientist class) that on escape method isnt allowed to escape. If it goes into escape zone it gives all the tickets to their team. In my case mtf. 

Expected outcome: 
Does not give tickets if they cant actually escape. 

Video:  https://medal.tv/games/scp-secret-laboratory/clips/19BYbWhNbp1Vnv/d13373BlLEy9?invite=cr-MSwzRkssNjYyODkyMjgs

### PR DESCRIPTION
**Pull request by 4310V343k**
also fixes #1810

### CODE REVIEW

**Comment by Misaka-ZeroTwo** [comment]
Should be getting tickets, not editing tickets.

File: Exiled.Events/EventArgs/Player/EscapingEventArgs.cs [hunk 1]
```diff
@@ -53,6 +87,11 @@ public EscapingEventArgs(Player player, RoleTypeId newRole, EscapeScenario escap
         /// </summary>
         public EscapeScenario EscapeScenario { get; set; }
 
+        /// <summary>
+        ///     Gets the TicketsToChange that will represent the amount of tickets granted after the player escapes.
+        /// </summary>
+        public Dictionary<SpawnableTeamType, float> TicketsToChange { get; }
```
Comment (Misaka-ZeroTwo) [inline Exiled.Events/EventArgs/Player/EscapingEventArgs.cs:93]: That's get only.

Comment (Misaka-ZeroTwo) [reply Exiled.Events/EventArgs/Player/EscapingEventArgs.cs:93]: Oh ok, then just do `get; set;`? Because that indicates the property can be edited.

**Review by louis1706 — CHANGES_REQUESTED** [review]
also there no need to make a Dictionary and new argument for the ctor

**Comment by louis1706** [comment]
this PR also conflict with #1807  (that not a requested change it's only informative)

**Review by louis1706 — APPROVED** [review]

**Comment by VALERA771** [comment]
Why you've created 2 new constructors?

**Pull request merged**